core/apps/ame-e2e/src/integration/editor/entity-that-extends-external-entity.cy.ts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2024 Robert Bosch Manufacturing Solutions GmbH
+ *
+ * See the AUTHORS file(s) distributed with this work for
+ * additional information regarding authorship.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+/// <reference types="Cypress" />
+
+import {SELECTOR_notificationsBtn, SELECTOR_notificationsDialogCloseButton} from '../../support/constants';
+
+describe('Test loading aspect with extended external Entity', () => {
```

Comment:
Include also same tests for different namespaces and maybe also for cross namespaces.

Reply:
done